app/controllers/time_summary_controller.rb

```diff
@@ -50,6 +50,17 @@ def update
     )
   end
 
+  def show
+    review = Review.find(params[:id]) unless params[:id].blank?
```

Comment:
```suggestion
    review = Review.find_by id: params[:id]
```
Funcionaría de la misma manera.